pxp_agent.yaml

```diff
@@ -0,0 +1,4 @@
+name: pxp-agent*
```

Comment:
Same, no idea how this is built. But AFAIK we are going to drop it because building this C++ tooling that has been unmaintained for years is a PITA. Correct?

Reply:
And this one should be from https://github.com/OpenVoxProject/pxp-agent-vanagon